pandas/_libs/interval.pyi

```diff
@@ -67,6 +65,8 @@ class Interval(IntervalMixin, Generic[_OrderableT]):
     def right(self: Interval[_OrderableT]) -> _OrderableT: ...
     @property
     def closed(self) -> IntervalClosedType: ...
+    mid: _MidDescriptor
+    length: _LengthDescriptor
```

Comment:
The current definition of `_MidDescriptor` and `_LengthDescriptor` still reference the case for `IntervalTree`. Might want to remove the `IntervalTree` case from the two descriptor and directly add it to `IntervalTree`.

Reply:
Oh, good point.  Responded to your earlier comment, and now realize this is a better way to go.